stable/external-dns/templates/configmap.yaml

```diff
@@ -3,8 +3,7 @@ apiVersion: v1
 kind: ConfigMap
```

Comment:
Idea for a future PR: be able to specify your external CA in an external CA.

Another question. The CA content is meant to be a public key, right? Otherwise it would make more sense to be a secret